plan/test_plan.py

```diff
@@ -585,98 +585,82 @@ def test_available_plans_for_sentry_plan_expired_trial_more_than_10_users(
 
 
 @freeze_time("2023-06-19")
-class AvailablePlansOngoingTrialMoreThanTenUsers(TestCase):
+class AvailablePlansOngoingTrial(TestCase):
     """
-    - users-trial, is trialing, more than 10 users -> users-pr-inappm/y, users-basic
-    - sentry-customer, users-trial, is trialing, more than 10 users -> users-pr-inappm/y, users-sentrym/y, users-basic
+    Non Sentry User is trialing
+        when <=10 activated seats -> users-pr-inappm/y, users-basic, users-litem/y
+        when > 10 activated seats -> users-pr-inappm/y, users-basic
+    Sentry User is trialing
+        when <=10 activated seats -> users-pr-inappm/y, users-sentrym/y, users-basic, users-litem/y
+        when > 10 activated seats -> users-pr-inappm/y, users-sentrym/y, users-basic
     """
 
     def setUp(self):
         self.current_org = OwnerFactory(
+            plan=PlanName.TRIAL_PLAN_NAME.value,
             trial_start_date=datetime.utcnow(),
             trial_end_date=datetime.utcnow() + timedelta(days=14),
             trial_status=TrialStatus.ONGOING.value,
-            plan_user_count=13,
+            plan_user_count=1000,
```

Comment:
nitpick: could we explicitly add plan_activated_users: None here? I like to initialize everything we use in our tests